File: app/src/main/java/com/callmangement/EHR/models/UpdateAttendanceStatusRoot.java
package com.callmangement.EHR.models;

public class UpdateAttendanceStatusRoot {
    public UpdateAttendanceStatusDatum Data;
    public String message;
    public String status;

    public UpdateAttendanceStatusRoot(UpdateAttendanceStatusDatum data, String message, String status) {
        Data = data;
        this.message = message;
        this.status = status;
    }


    public UpdateAttendanceStatusDatum getData() {
        return Data;
    }

    public void setData(UpdateAttendanceStatusDatum data) {
        Data = data;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/UpdateAttendanceStatusRoot.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName

class UpdateAttendanceStatusRoot(
    @SerializedName("Data")
    var data: UpdateAttendanceStatusDatum,
    @JvmField var message: String,
    @JvmField var status: String
)
